runtime/src/main/java/io/novafoundation/nova/runtime/util/FullAssetIdModel.kt

```diff
@@ -0,0 +1,47 @@
+package io.novafoundation.nova.runtime.util
+
+import android.os.Parcel
+import android.os.Parcelable
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.ChainId
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.FullChainAssetId
+
+class FullAssetIdModel(val chainId: ChainId, val assetId: Int) : Parcelable {
+
+    constructor(parcel: Parcel) : this(
```

Comment:
Cant we use Parcelize here?

Reply:
Of course. We just need to add `kotlin-android-extensions` plugin